sdk/communication/AzureCommunicationChat/Source/Signalling/events/chat.swift

```diff
@@ -0,0 +1,304 @@
+//
+//  chat.swift
+//  AzureCommunicationSignaling
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AzureCommunication
+import AzureCore
+import Foundation
+
+// TODO: Can we use existing models?
+public class SignallingChatParticipant {
+    public var id: CommunicationIdentifier?
+    public var displayName: String?
+    public var shareHistoryTime: Iso8601Date?
+
+    init(id: CommunicationIdentifier?, displayName: String? = nil, shareHistoryTime: Iso8601Date? = nil) {
+        self.id = id
+        self.displayName = displayName
+        self.shareHistoryTime = shareHistoryTime
+    }
+}
+
+public class SignallingChatThreadProperties {
+    public var topic: String
+
+    init(topic: String) {
+        self.topic = topic
+    }
+}
+
+public class BaseChatEvent {
+    public var threadId: String
+    public var sender: CommunicationIdentifier?
+    public var senderDisplayName: String?
+    public var recipient: CommunicationIdentifier?
+
+    init(
+        threadId: String,
+        sender: CommunicationIdentifier?,
+        senderDisplayName: String? = nil,
+        recipient: CommunicationIdentifier?
+    ) {
+        self.threadId = threadId
+        self.sender = sender
+        self.senderDisplayName = senderDisplayName
+        self.recipient = recipient
+    }
+}
+
+public class BaseChatThreadEvent {
+    public var threadId: String
+    public var version: String
+
+    init(threadId: String, version: String) {
+        self.threadId = threadId
+        self.version = version
+    }
+}
+
+public class BaseChatMessageEvent: BaseChatEvent {
+    public var id: String
+    public var createdOn: Iso8601Date?
+    public var version: String
+
```

Comment:
"type" is missing here.

Type of the chat message. Possible types are "Text" and "RichText/Html".

Reply:
yes, `type` seems missing here.